src/utils/formatters.ts

```diff
@@ -5,17 +5,7 @@ import { formatAmount } from './formatNumber'
 const GWEI = 'gwei'
 
 export const _removeTrailingZeros = (value: string): string => {
-  let [integer, fractions] = value.split('.')
-
-  if (!fractions) {
-    return value
-  }
-
-  while (fractions[fractions.length - 1] === '0') {
-    fractions = fractions.substring(0, fractions.length - 1)
-  }
-
-  return fractions ? `${integer}.${fractions}` : integer
+  return value.replace(/(\.0*|(?<=(\..*))0*)$/, '')
```

Comment:
This regex looks complicated. Can you explain what it matches?
If you just need to trim a dot and zeros at the end, `/\.0+$/` should do the trick.

Reply:
We don't want to just trim a dot and zeros but also lookbehind the dot and any non-zero decimals for trailing zeros as well, e.g. `0.001000` => `0.001`.

It's taken from [here](https://stackoverflow.com/a/65172703/6585368). Is adding a summarised comment sufficient?

Reply:
Ah, gotcha. A dot somewhere before the zeros but not necessary directly adjacent.